examples/rc-cpy-final.py

```diff
@@ -14,182 +15,163 @@
 from digitalio import DigitalInOut, Direction
 from pulseio import PWMOut, PulseIn, PulseOut
 
-## Customisation these variables
+import adafruit_logging as logging
+logger = logging.getLogger('code')
+logger.setLevel(logging.INFO)
+
+# Customisation these variables
 SMOOTHING_INTERVAL_IN_S = 0.025
 DEBUG = False
 ACCEL_RATE = 10
 
 ## functions
 def servo_duty_cycle(pulse_ms, frequency = 60):
-	"""
-	Formula for working out the servo duty_cycle at 16 bit input
-	"""
-	period_ms = 1.0 / frequency * 1000.0
-	duty_cycle = int(pulse_ms / 1000 / (period_ms / 65535.0))
-	return duty_cycle
+    """
+    Formula for working out the servo duty_cycle at 16 bit input
+    """
+    period_ms = 1.0 / frequency * 1000.0
+    duty_cycle = int(pulse_ms / 1000 / (period_ms / 65535.0))
+    return duty_cycle
+
 
 def state_changed(control):
-	"""
-	Reads the RC channel and smooths value
-	"""
-        prev = control.value
-	control.channel.pause()
-	for i in range(0, len(control.channel)):
-		val = control.channel[i]
-                # prevent ranges outside of control space
-		if(val < 1000 or val > 2000):
-			continue
-		# set new value
-		control.value = (control.value + val) / 2
-
-	if DEBUG:
-		print("%f\t%s (%i): %i (%i)" % (time.monotonic(), control.name, len(control.channel), control.value, servo_duty_cycle(control.value)))
-	control.channel.clear()
-	control.channel.resume()
-
-def state_changed_throttle(control):
-	"""
-	(Testing) Reads RC channel, smooths value and stall prevention (disabled)
-	"""
-        prev = control.value
-	control.channel.pause()
-	for i in range(0, len(control.channel)):
-		#val = control.channel[i]
-                # prevent ranges outside of control space
-		if(val < 1000 or val > 2000):
-			continue
-		# cap maximum acceleration to prevent stall
```

Comment:
I would like to keep this algorithm somewhere, even if it is disabled as it is useful.

Reply:
this is not changed i believe. maybe just re-formatting causing it looks like it is deleted.